misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/OptionalAssertionsSpec.kt

```diff
@@ -10,8 +10,9 @@ import org.spekframework.spek2.style.specification.Suite
 import java.lang.AssertionError
 import java.util.Optional
 
-abstract class OptionalAssertionsSpec (
+abstract class OptionalAssertionsSpec(
     isEmpty: Fun0<Optional<Any>>,
+    isPresent: Fun1<Optional<Any>, Any>,
     describePrefix: String = "[Atrium] "
 ) : Spek({
```

Comment:
Please add your functions to SubjectLessSpec as well (see ResultFeatureAssertionsSpec) + add AssertionCreatorSpec (I think you can copy most of the code from ResultFeatureAssertionsSpec)